src/js/productList.mjs

```diff
@@ -3,27 +3,47 @@ import { renderListWithTemplate } from "./utils.mjs";
 
 function productCardTemplate(product) {
   return `<li class="product-card">
-            <a href="product_pages/index.html?product=${product.Id}">
-              <img
-                src="${product.Image}"
-                alt="Image of ${product.Name}"
-              />
-              <h3 class="card__brand">${product.Brand.Name}</h3>
-              <h2 class="card__name">${product.Name}</h2>
-              <p class="product-card__price">$${product.FinalPrice}</p></a>
-          </li>`; 
+      <a href="product_pages/index.html?product=${product.Id}">
+      <img
+      src="${product.Image}"
+      alt="Image of ${product.Name}"
+      />
+      <h3 class="card__brand">${product.Brand.Name}</h3>
+      <h2 class="card__name">${product.Name}</h2>
+      <p class="product-card__price">$${product.FinalPrice}</p></a>
+      </li>`;
```

Comment:
Cleaned this function up so that it remains just a function for converting a product to HTML